openmdao/docs/openmdao_book/features/core_features/running_your_models/set_input_defaults.ipynb

```diff
@@ -0,0 +1,475 @@
+    "        :noindex:\n",
+    "```\n",
+    "\n",
+    "This function does not set the actual values of the inputs, only the metadata that will be used to populate the [AutoIVC](../../../other_useful_docs/auto_ivc_api_translation.ipynb) output connected to them. The metadata specified via `set_input_defaults` is applied during the model setup phase, when the AutoIVC connections are resolved.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example #1\n",
+    "\n",
+    "In this example, we have two components that promote the variable `x` but use different units.  This will result in an error during `setup` if left unresolved:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openmdao.api as om\n",
+    "\n",
+    "p1 = om.Problem()\n",
+    "model = p1.model\n",
+    "\n",
+    "# Note that units and value to use for the promoted variable 'x' are ambiguous due to having different defaults\n",
+    "model.add_subsystem('C1', om.ExecComp('y = 3.*x', x={'val': 3000., 'units': 'mm'}), promotes=['x'])\n",
+    "model.add_subsystem('C2', om.ExecComp('y = 4.*x', x={'val': 400., 'units': 'cm'}), promotes=['x'])\n",
+    "\n",
+    "try:\n",
+    "    p1.setup()       \n",
+    "except Exception as err:\n",
+    "    print(str(err))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "remove-input",
+     "remove-output",
+     "allow-assert"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    p1.setup()       \n",
+    "except Exception as err:\n",
+    "    assert(str(err) == \"\\nCollected errors for problem 'problem':\\n   <model> <class Group>: The following inputs, ['C1.x', 'C2.x'], promoted to 'x', are connected but their metadata entries ['units', 'val'] differ. Call <group>.set_input_defaults('x', units=?, val=?), where <group> is the model to remove the ambiguity.\")\n",
+    "else:\n",
+    "    raise RuntimeError(\"Exception expected.\")    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ambiguity can be resolved by setting the desired default values for the promoted variable at the group level by calling `set_input_defaults`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "model.set_input_defaults('x', val=1., units='m')\n",
+    "\n",
+    "# Note that the we have specified the default metadata for the AutoIVC output that will supply values\n",
+    "# for the promoted 'x' (_auto_ivc.v0) but it has not been applied yet. It will be applied during setup.\n",
+    "\n",
+    "# Calling list_inputs at this point will show the default values from the component definitions\n",
+    "model.list_inputs(units=True)\n",
+    "model.list_outputs(units=True, list_autoivcs=True);"
```

Comment:
I wonder if using `list_vars` instead of `list_inputs` and `list_outputs` is more convenient for the user.